tests/csrf_tests/tests.py

```diff
@@ -237,7 +237,7 @@ def test_token_node_no_csrf_cookie(self):
         """
         req = self._get_GET_no_csrf_cookie_request()
         resp = token_view(req)
-        self.assertEqual(resp.content, b'')
+        self.assertNotEqual("", resp.content)
```

Comment:
This change turns this into a confusing/possibly-useless test. What is the response content? What assertion can we make about it? The universe of "things which are not the empty string" is very large, and includes many things which would be wrong.